### ISSUE TITLE
Bughunting: mjsunit/regress/regress-165637 fails on MIPS64 BE

### ISSUE THREAD

**Issue by mkaric** [v8mips/v8mips#363]
Log:
https://v8.mips.com/bbv8/builders/Target%20-%20MIPS64%20-%20big%20-%20Debug%20-%20edge%20-%20no%20i18n/builds/1192/steps/Check%20%28flakes%29/logs/regress-165637

**Comment by mkaric** [comment]
On buildbots, this test started failing from the CL https://codereview.chromium.org/2156573002 (commit 2aa2b65). When I test this commit, the test fails, but previous commits also fail.

The test does array slices in a loop and measures that the time to finish is less than 3 seconds. On QEMU I get about 15 seconds and 10 seconds on a BE board. That is similar to buildbots, but on the second run the test passes.

**Comment by mkaric** [comment]
When I run the test on a LE mips64 board, the test runs for about 8 seconds and fails. This board is faster than the BE one I used previously, so slightly faster test execution was expected. Then I tried a faster BE board and the test runs for about 5 seconds.

My guess is the test execution time is around 3 seconds on the buildbot, so it started failing without significant change in the code, just as it now started passing.

I'll wait a bit and close the issue if the tests continues passing. If this comes up again, maybe we could add a patch on the buildbot that would print the execution time so we can see if there is a significant difference when the test passes.
